templates/vue-demo-store/components/checkout/cart/ProductItem.vue

```diff
@@ -92,7 +99,11 @@ const removeCartItem = async () => {
       <div
         class="flex flex-col lg:flex-row justify-between text-base font-medium text-secondary-900"
       >
-        <h3 class="text-base" data-testid="cart-product-name">
+        <h3
+          class="text-base cursor-pointer"
+          data-testid="cart-product-name"
+          @click="navigateToItemDetailPage(cartItem.id)"
+        >
```

Comment:
It should be wrapped with NuxtLink + additional @click event. Otherwise causes accessibility issue. Look inside LayoutSideMenu component for the inspiration

Reply:
I've implemented the functionality to close the miniCartModal when the NuxtLink click event occurs. Please review the changes and let me know if anything needs adjustment. Thank you!